shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -123,6 +123,10 @@ TEST_F(EmbedderTest, CanInvokeCustomEntrypointMacro) {
   ASSERT_TRUE(engine.is_valid());
 }
 
+/*
+ * Uncomment once a Dart roll with the fix to reinitialize dart-io state
```

Comment:
nit: add a link to the dart CL so we know which one.

Reply:
Done.